src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {

```

Comment:
According to the checklist, there shouldn't be an empty line at the beginning of a class definition. Please remove the empty line before the class declaration.